ControlMeasurements/ControlMeasurements.csproj

```diff
@@ -2,6 +2,9 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
+    <AssemblyName>p</AssemblyName>
```

Comment:
Dodawales te 3 linijki specjalnie, czy same sie dodaly? Jesli same, to je wywal i zobacz czy dalej bedzie dzialac :)

Reply:
Same sie dodaly. Zostawiłem je bo jak wywalam to nie działa :)

Reply:
Aha ok :D